synapse/lib/storm.py

```diff
@@ -1397,3 +1398,21 @@ def _stormOperGuid(self, query, oper):
         core = self.getStormCore()
 
         [query.add(node) for node in core.getTufosByIdens(args)]
+
+    def _stormOperTask(self, query, oper):
```

Comment:
This looks good so far. We had a little discussion about it. My concerns were that: if the task originator goes away, nothing is there to clean up the tags, if a task runner dies while running a task, the task is either lost, or would be pulled again (and probably break it again). These were deemed out of scope for now.  I think if we're going to merge it before the persistent queue is ready, we should add a logger warning that says something like "this will not actually do anything yet".